File: app/src/main/java/net/sunniwell/aidl/bean/AreaBean.java
package net.sunniwell.aidl.bean;

import org.xmlpull.v1.XmlPullParser;

import java.io.Serializable;

public class AreaBean implements Serializable {
    private static final long serialVersionUID = 1;

    /* renamed from: id */
    private int f298id = 0;
    private String title = XmlPullParser.NO_NAMESPACE;

    public void setId(int id) {
        this.f298id = id;
    }

    public int getId() {
        return this.f298id;
    }

    public void setTitle(String title2) {
        this.title = title2;
    }

    public String getTitle() {
        return this.title;
    }
}

File: app/src/main/java/com/eliving/tv/aidl/bean/AreaBean.kt
package com.eliving.tv.aidl.bean

import org.xmlpull.v1.XmlPullParser
import java.io.Serializable

class AreaBean : Serializable {
    /* renamed from: id */
    var id = 0
    var title = XmlPullParser.NO_NAMESPACE

    companion object {
        private const val serialVersionUID: Long = 1
    }
}